src/components/parts/contact-option/ContactOptionPart.tsx

```diff
@@ -94,9 +94,9 @@ export const ContactOptionPart = ({ config }: ContactOptionProps) => {
                     bem('icon', selectedChannel)
                 )}
             />
-            <Undertittel tag="h3" className={bem('title')}>
+            <Title level={2} size="s" className={bem('title')}>
```

Comment:
Bør kanskje opp til m?

Reply:
Den vipper mellom 2 størrelser nemlig... S blir for lite og m blir for stort. Jeg valgte s for å unngå at alle overskrifter og titler ble så himla store...